dlc2nwb/utils.py

```diff
@@ -24,7 +62,7 @@ def get_movie_timestamps(movie_file, VARIABILITYBOUND=1000):
     """
     # TODO: consider moving this to DLC, and actually extract alongside video analysis!
 
-    reader = VideoReader(movie_file)
+    reader = cv2.VideoCapture(movie_file)
     timestamps = []
```

Comment:
For this specific case all the machinery of your native VideoReader does not seem necessary as we are only extracting the timestamps if available.

Reply:
That makes sense!

Reply:
> > I was looking at your configuration files (`setup.py` and `setup.cfg`) and I did not know what standard do you use for writing your dependencies. In the `deeplabcut` repo you seem to use a `requirements.txt` fule but also have them on the `setup.py`. Let me know what standard you use and I can write the dependencies of the repo there.
> 
> setup.cfg is the one :)

@jeylau
I add the dependencies over there then. Also, added `deeplabcut` to the testing in extras_require.